packages/paste-design-tokens/tokens/themes/dark/global/background-color.yml

```diff
@@ -28,11 +28,20 @@ props:
 
   # brand backgrounds
   color-background-brand:
-    value: "{!palette-blue-80}"
+    value: "{!palette-gray-80}"
     comment: Primary brand background, accessible with inverse text
+  color-background-brand-strong:
+    value: "{!palette-gray-90}"
+    comment: Strong primary brand background, accessible with inverse text
+  color-background-brand-stronger:
+    value: "{!palette-gray-100}"
+    comment: Stronger primary brand background, accessible with inverse text
   color-background-brand-highlight:
-    value: "{!brand-highlight}"
+    value: "{!amaranth}"
     comment: Highlight brand background, accessible with black text
+  color-background-brand-highlight-lightest:
+    value: "{!amaranth-transparent-10}"
+    comment: Lightest highlight brand background, accessible with black text
```

Comment:
A bunch of other token comments still say "light"/"dark" instead of "strong"/"weak" https://deploy-preview-1434--paste-docs.netlify.app/tokens

Reply:
Unrelated thing:

The border width comments seem to randomly say "Large border width" and "Small border width".